server/controllers/events/events_controller_e2e_test.go

```diff
@@ -42,7 +42,7 @@ import (
 	. "github.com/runatlantis/atlantis/testing"
 )
 
-const ConftestVersion = "0.35.0"
+const ConftestVersion = "0.35.0" // renovate: datasource=github-releases depName=open-policy-agent/conftest
```

Comment:
This test for the exact conftest version in the e2e tests seems silly to me and it's difficult to upgrade conftest because it always fails the test. There is a chicken and egg issue created with this test because the test relies on a new testing-env container which cannot be created in the same PR as this line getting modified.

We also have to setup renovatebot to update the testing environment 

https://github.com/runatlantis/atlantis/blob/6a7f79e749a1604f54378e4920cf83c2888399ec/.github/workflows/test.yml#L30

https://github.com/runatlantis/atlantis/blob/6a7f79e749a1604f54378e4920cf83c2888399ec/.circleci/config.yml#L5

Instead, it would be better to do the following

- get rid of chicken-and-egg: Change this test to verify that the version is greater than ConftestVersion. This could be bundled with this PR or in a separate one. But this PR as-is will still make it hard to approve and merge when tests are failing.
- add the terraform installs and conftest installs to the base image, rm the testing-env image, and run tests on the base image. This would be more effort and probably done in a separate PR.

Reply:
I see, you are right. Current conftest update PR may fail CI, cause testing-env doesn't have `contest$nextVersion` command. https://github.com/krrrr38/atlantis/pull/12/files

> get rid of chicken-and-egg: Change this test to verify that the version is greater than ConftestVersion. This could be bundled with this PR or in a separate one. But this PR as-is will still make it hard to approve and merge when tests are failing.

I will create following PRs.

1. testing-env allow to use `contest` command, not `conftest$version` by edit here https://github.com/runatlantis/atlantis/blob/6a7f79e749a1604f54378e4920cf83c2888399ec/testing/Dockerfile#L25
2. after update testing-env docker image, change [e2e ensure logic](https://github.com/runatlantis/atlantis/blob/6a7f79e749a1604f54378e4920cf83c2888399ec/server/controllers/events/events_controller_e2e_test.go#L1368) to use `conftest` instead of `contest$version`
3. conftest update flow need following 3PR. So change renovate setting to make different `i` and `iii` PRs.
   1. First, update `testing/Dockerfile` conftest version.
   2. Second, update `testing-env` docker image tag
   3. Third, update otherwise contest versions, which contains `Dockerfile`, `Dockerfile.dev` and `events_controller_e2e_test.go`

Reply:
I added `packageRules` to separate the following PRs in https://github.com/runatlantis/atlantis/pull/2807/commits/fe8dad4040dc651269b26a24a6653ae1417e9c07.

- https://github.com/krrrr38/atlantis/pull/29
  - upgrade testing/Dockerfile conftest version.
- https://github.com/krrrr38/atlantis/pull/28
  - upgrade otherwise conftest versions.